Formula/tcl-tk.rb

```diff
@@ -1,11 +1,10 @@
 class TclTk < Formula
   desc "Tool Command Language"
-  homepage "https://www.tcl.tk/"
```

Comment:
Is this documented somewhere?

Reply:
If you look at the footer on https://www.tcl.tk, it says "This is the main Tcl Developer Xchange site, www.tcl-lang.org", seemingly pointing to that domain being the more canonical one now. Additionally any links to https://core.tcl.tk automatically redirect to https://core.tcl-lang.org. I could probably find more evidence if this isn't sufficient.

If it matters, MacPorts [switched to the tcl-lang.org domain](https://github.com/macports/macports-ports/commit/0b46a81e53cdb735d6b3704d4ac51d0387bad3f2) on 2019/06/06.

Reply:
Right, I checked for a redirect but didn't spot the text in the footer